projects/canopy-test-app/src/app/app.component.html

```diff
@@ -1395,4 +1403,4 @@ <h3>Carousel item 3</h3>
   <footer lg-footer>
     <lg-footer-logo src="assets/legal-and-general-logo.svg"></lg-footer-logo>
   </footer>
-</lg-page>
+</lg-page>
```

Comment:
surprised the linter didn't pick this up - could you add a new line here, please?

Reply:
it's the end of the file, with no newline at the end

Reply:
Done. Will push the changes shortly. Not sure why prettier didn't automatically format it either.